src/services/database/schema/v2.js

```diff
@@ -0,0 +1,244 @@
+// @flow
+// Schema v2: Effective from 0.3.3 to Current
+const uString = require('unique-string');
```

Comment:
Can we use import here instead of require? It gives more consistency with other project parts.

Reply:
done